README.md

```diff
@@ -28,12 +28,16 @@ Below are examples of the dice roll syntax.
 
 `!roll 3d6 e6` : Roll three six-sided dice and explode on sixes. Some game systems call this 'open eneded' dice. If the number rolled is greater than or equal to the value given for this option, the die is rolled again and added to the total. If no number is given for this option, it is assumed to be the same as the number of sides on the die. Thus, '3d6 e' is the same as '3d6 e6'.
 
+`!roll 3d6 e6` : Roll three six-sided dice and explode on sixes indefinitly. 
+
 `!roll 3d10 d1` : Roll three ten-sided dice and drop one die. The lowest value will be dropped first.  **NOTE:** These dice are dropped before any dice are kept with the following `k` command. Order of operations is : roll dice, drop dice, keep dice
 
 `!roll 3d10 k2` : Roll three ten-sided dice and keep two. The highest value rolled will be kept. 
 
 `!roll 4d6 r2` : Roll four six-sided dice and reroll any that are equal to or less than two. 
```

Comment:
Should this be updated now to say something like "reroll once any that are equal to or less than two"?